pacemaker.spec.in

```diff
@@ -267,7 +269,10 @@ Available rpmbuild rebuild options:
 License:       GPLv2+ and LGPLv2+
 Summary:       Command line tools for controlling Pacemaker clusters
 Group:         System Environment/Daemons
-Requires:      %{name}-libs = %{version}-%{release}
+Requires:      %{name}-libs%{?_isa} = %{version}-%{release}
+%if 0%{?fedora} > 22 || 0%{?rhel} > 7
+Recommends:    pcmk-cluster-manager = %{version}-%{release}
+%endif
```

Comment:
Re first:

Subjective rather unfounded opinion based on the idea that it was added
initially for simplicity and easy testability by the means of CTS,
I admit.  But given that such a feature is (intentionally I presume,
otherwise pcs would be driven to utilize it, e.g. for pcsd-less
cluster discovery, but that would require it would have to be
originally enabled, like by pcs itself, so that makes a chicken/egg)
kept in the shadow makes it sort of an exotic one in my book.

* * *

Re second:

Oops, I've probably missed that, but supposing only `-cli` users where
`Recommends:` apply will want to `Requires:` this `-cli` (full-fledged
`pacemaker` or `pcmk-cluster-manager` otherwise), it's an non-issue.

* * *

> It would be nice (if possible) to add a preference for the main
> pacemaker package over pacemaker-remote if someone tries to install
> just -cli.

It's left to the actual `-cli` dependants in the envisioned scheme.
Consider hypothetical `pcs-remote` package.

> I would discourage anyone from making other packages depend solely
> on -cli at this time.

No regression here, the situation is actually getting better for
when weak dependencies are available, but since it's distro-contained
thing as mentioned, the dependants in other distributions need to
depend more wisely than on `-cli` (again, no regression, just need
to realize that this change cannot be leveraged universally).